src/components/card/card.stories.mdx

```diff
@@ -528,6 +532,113 @@ Click on the card to see the effect
   </Story>
 </Preview>
 
+## With draggable
+
+<Preview>
+  <Story
+    name="with draggable"
+    parameters={{ chromatic: { disable: true } }}
+  >
+  <Card draggable>
+      <CardRow>
+        <CardColumn align="left">
+          <Heading title="Stripe - [account name]" divider={false} />
+          <Typography variant="h5">user.name@sage.com</Typography>
+        </CardColumn>
+        <CardColumn align="right">
+          <Icon type="image" />
+        </CardColumn>
+      </CardRow>
+      <CardRow>
+        <CardColumn>
+          <Typography variant="h5" weight="bold">
+            Stripe Balance
+          </Typography>
+          <Heading title="£ 0.00" divider={false} />
+          <Typography>LAST ENTRY: 15 DAYS AGO</Typography>
+        </CardColumn>
+      </CardRow>
+      <CardFooter>
+        <CardColumn>
+          <Link icon="link" href="https://carbon.sage.com/">
+            View Stripe Dashboard
+          </Link>
+        </CardColumn>
+      </CardFooter>
+    </Card>
+  </Story>
+</Preview>
+
+### As a Draggable container
+
+<Preview>
+  <Story
+    name="with draggable conatiner"
+    parameters={{ chromatic: { disable: true } }}
+  >
+    <DraggableContainer>
+      <DraggableItem key="1" id={1}>
+        <Card>
+          <CardRow>
+            <CardColumn align="left">
+              <Heading title="Stripe - [account name]" divider={false} />
+              <Typography variant="h5">user.name@sage.com</Typography>
+            </CardColumn>
+            <CardColumn align="right">
+              <Icon type="image" />
+            </CardColumn>
+          </CardRow>
+          <CardRow>
+            <CardColumn>
+              <Typography variant="h5" weight="bold">
```

Comment:
I think we should make these two cards different so you can see they actually changed position.

Reply:
Good point. I'll make sure they are different 😃